app/helpers/hyrax_helper.rb

```diff
@@ -21,6 +21,10 @@ def banner_image
     Site.instance.banner_image? ? Site.instance.banner_image.url : super
   end
 
+  def favicon
+    Site.instance.favicon? ? Site.instance.favicon.url : super
+  end
+
```

Comment:
@labradford should this PR only hide the tab, or is this needed here?

Reply:
i want to see if that code fixes the bug we are experiencing